app/reducers/optional.js

```diff
@@ -1,7 +0,0 @@
-//this is a simple pass thru that offers an extension point
```

Comment:
@ksnyde if you look at the documentation site under configure you can see my mention (along w/ plans to remove it likely in a 2x branch) :)

http://www.ember-redux.com/configure/